lib/file-utilities.js

```diff
@@ -205,35 +206,20 @@ function streamAllLogs(config, callback) {
     var journalctl, lastchunk, i
 
     if (config.usejournald === 'files') {
-      journalctl = spawn('journalctl', ['-n0', '-D', '/var/log/journal', '-o', 'json', '-f'])
+      journalctl = spawn(
+        'journalctl',
+        ['-n0', '-D', '/var/log/journal', '-o', 'json', '-f']
+      )
     } else {
       journalctl = spawn('journalctl', ['-n0', '-o', 'json', '-f'])
     }
 
-    const processChunk = (data) => {
-      try {
-        data = JSON.parse(data)
-      } catch (e) {
-        return {t: Date.now(), l: data, f: 'systemd'}
-      }
-      if (data.__REALTIME_TIMESTAMP && parseInt(data.__REALTIME_TIMESTAMP) > 10000000000000) {
-        // convert to ms
-        data.__REALTIME_TIMESTAMP = parseInt(data.__REALTIME_TIMESTAMP / 1000)
-      }
-      return {
-        t: data.__REALTIME_TIMESTAMP
-      , line: data.MESSAGE
-      , f: data.CONTAINER_NAME || data._SYSTEMD_UNIT || data.SYSLOG_IDENTIFIER || 'UNKNOWN_SYSTEMD_APP'
-      , pid: data._PID && parseInt(data._PID)
-      , prival: data.PRIORITY && parseInt(data.PRIORITY)
-      , containerid: data.CONTAINER_ID_FULL
-      }
-    }
-
     journalctl.stdout.on('data', (data) => {
       data = data.toString().trim().split('\n')
       for (i = 0; i < data.length; i++) {
-        if (data[i].substring(0, 1) === '{' && data[i].substring(data[i].length - 1) === '}') {
+        if (data[i].substring(0, 1)
+          === '{' && data[i].substring(data[i].length - 1)
+          === '}') {
```

Comment:
nit picky, but put each conditional on its own line. when its split like this it can be difficult to understand where one check starts and one check stops.

```javascript
if (
  data[i].substring(0, 1)=== '{' 
  && data[i].substring(data[i].length - 1)=== '}'
) {
```

 it would also make it a bit more obvious to pull the data[i] bits out in to a variable

```javascript
const start = data[i].substring(0, 1)
const end = data[i].substring(data[i].length - 1)

if (start === '{' && end === '}') {
  ...
}
```